examples/compute_intensive/run_test

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+rm -fr .ert
```

Comment:
Remember to remove this. We are soon introducing the possibility to remove experiments and that would be the correct approach if the experiment have already been ran...